### ISSUE TITLE
The spacing is 1.X instead of 1.5

### ISSUE THREAD

**Issue by akmalirfan** [utmthesis/utmthesis#11]
Even though the spacing is defined with \onehalfspacing in utmthesis.cls, the spacing in the main body still appear as 1 instead of 1.5.

At first, I thought it is only my problem since I am using many packages which may introduce some conflicts. Then, I downloaded the PDF file in this repository and compared its chapter 1 to mine.

After some googling, I think it is because of the conflict between setspace and hyperref. Refer to [this comment](https://texblog.org/2011/09/30/quick-note-on-line-spacing/#comment-5654). The author of the article suggests loading hyperref as the last package. I tried to do it myself but I failed.

**Comment by yungszen** [comment]
That's strange. So far, we never had a problem with the line spacing.  
Which version are you using?

**Comment by akmalirfan** [comment]
I'm using 4.1 but the problem is still reproducible in the current version (5.1). To prove that, I downloaded the current version. Using MiKTeX on Windows 10, I compiled it twice (because that's how things work sometimes in TeX world). Then, I compared it to a PDF file that I exported using a MS Word template of UTM thesis. Here are the two documents side by side:

![image](https://user-images.githubusercontent.com/5904789/27009089-9201d4f8-4eb7-11e7-85bf-c9302cf541a9.png)

I have double-checked the spacing in Word and it is 1.5. I also compared the PDF file that I compiled with the PDF file that comes with this repository initially and they have the same spacing. I'm not sure whether it is 1.0 or 1.15 or else but it certainly is less than 1.5.

**Comment by yungszen** [comment]
I've double checked this and you are right.  Not sure when this problem started. 
Perhaps you can try \doublespacing and see if it works.   You would need to change several locations in the utmthesis.cls

**Comment by akmalirfan** [comment]
I asked [mastermath](https://www.fiverr.com/mastermath) over Fiverr to have a look into this matter. He suggested changing all occurrences of \onehalfspacing to \setstretch{1.5}. I tried his suggestion and found out that 1.5 is a bit too much. Through trial and error, I found out that 1.435 produces the best output. Below is the screenshot of my finding:

![buang](https://user-images.githubusercontent.com/5904789/27210022-114accb0-5282-11e7-8c7d-3053919d8fe2.png)

The PDF file on the foreground is the problematic one. In the smaller window is the fixed one. In the smallest window is the one that I produced using MS Word.

I am going to submit a pull request now.
